ntbs-service/appsettings.Development.json

```diff
@@ -38,7 +39,8 @@
     "TablePrefix": "Dev"
   },
   "ScheduledJobsConfig": {
-    "UserSyncEnabled": false,
+    "UserSyncEnabled": true,
+    "UserSyncCron": "30 3 * * 1",
```

Comment:
I think we should probably leave these jobs disabled by default, and only enable them when we're actually working on them.

Reply:
That makes sense! Removed in daa98be 🙂